L/libfido2/build_tarballs.jl

```diff
@@ -0,0 +1,59 @@
+using BinaryBuilder
+using Pkg: PackageSpec
+name    = "libfido2"
+version = v"1.16.0"
+
+sources = [
+    ArchiveSource(
+        "https://developers.yubico.com/libfido2/Releases/libfido2-$(version).tar.gz",
+        "8c2b6fb279b5b42e9ac92ade71832e485852647b53607c43baaafbbcecea04e4",
+        unpack_target="src",
+    ),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/src/libfido2-*
```

Comment:
It seems here is no need for a custom `unpack_target`, or is there?

So you could do
```suggestion
    ArchiveSource(
        "https://developers.yubico.com/libfido2/Releases/libfido2-$(version).tar.gz",
        "8c2b6fb279b5b42e9ac92ade71832e485852647b53607c43baaafbbcecea04e4"
    ),
]

script = raw"""
cd ${WORKSPACE}/srcdir/libfido2-*
```
or even follow the convention used in many other build recipes:
```suggestion
    ArchiveSource("https://developers.yubico.com/libfido2/Releases/libfido2-$(version).tar.gz",
                  "8c2b6fb279b5b42e9ac92ade71832e485852647b53607c43baaafbbcecea04e4"),
]

script = raw"""
cd ${WORKSPACE}/srcdir/libfido2-*
```